content/en/Getting started/planning.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Set Up Pentest Planning"
+linkTitle: "Set Up Pentest Planning"
+weight: 210
+description: >
+  Set up a schedule. Confirm the scope.
+---
+
+{{% pageinfo %}}
+Now you can set a date for the pentest.
+{{% /pageinfo %}}
+
+By default, we can help you schedule pentests with a start date at least
+48 hours after you select "Submit for Review".
```

Comment:
We note the start date limited to "2 business days" in the application, which I believe is in line with our staffing process.  For example, there is form validation in the wizard Planning step: "The pentest start date needs to be at least 2 business days from today,"

Reply:
Yes, for consistency with language in the UI (when it's clear)